tools/plugins/com.liferay.ide.gradle.ui/plugin.xml

```diff
@@ -42,6 +42,21 @@
                class="org.eclipse.core.resources.IResource">
          </selection>
       </wizard>
+      <wizard
```

Comment:
This should not be listed in the "new" wizards.  Instead there should be an "action" that is added to the context menu of an existing fragment project (or fragment bnd.bnd file).